checks.d/network.py

```diff
@@ -176,6 +176,8 @@ def _check_linux(self, instance):
                 metrics = self._parse_linux_cx_state(lines[2:], self.TCP_STATES['netstat'], 5)
                 for metric, value in metrics.iteritems():
                     self.gauge(metric, value)
+            except Exception:
+                self.log.exception("Error collecting connection stats.")
```

Comment:
log.exception() already adds the exception info - type, stacktrace to the logging statement.

I have added the custom exception - for future use, but I don't think it's necessary to go over all try...catch blocks that already handler `Exception` and add yet another block that basically will do nothing other than log the exception. Sounds like overkill. But that's just me, coming from a C/C++ background.